sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -203,6 +203,24 @@ def test_rfecv_mockclassifier():
     assert_equal(len(rfecv.ranking_), X.shape[1])
 
 
+def test_rfecv_verbose_output():
+    """Check verbose=1 is producing an output."""
```

Comment:
Please use a comment, not a docstring. That makes it easier to find which test was run using nosetests, which we are (still) using.